cmd/main.go

```diff
@@ -137,17 +133,10 @@ func main() {
 		rep.IncOpsSeen(),
 	)
 
-	b := bundler.New(mem, chain, values.SupportedEntryPoints)
-	b.SetGetBaseFeeFunc(gasprice.GetBaseFeeWithEthClient(eth))
-	b.SetGetGasTipFunc(gasprice.GetGasTipWithEthClient(eth))
-	b.SetGetLegacyGasPriceFunc(gasprice.GetLegacyGasPriceWithEthClient(eth))
-	b.UseLogger(logger)
-	if err := b.UserMeter(otel.GetMeterProvider().Meter("bundler")); err != nil {
-		log.Fatal(err)
-	}
+	bundlerClient := createBundlerClient(mem, chain, values, eth, stdLogger)
 
 	var check = (*checks.Standalone)(unsafe.Pointer(validator))
```

Comment:
I'm concerned about line #138 I wrote because there's no compiler validation validating binary compatibility moving forward. 
I see 2 options
Make a safe type cast and panic if not successful 
or
copying the state of 'validator' to 'check'
What do you think?